wasm-runtime/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "wasm-runtime"
+version = "0.1.0"
+authors = ["Stefan Junker <mail@stefanjunker.de>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+wasm-libruntime = { path = "../wasm-libruntime" }
+
+env_logger = "0.7"
+log = "0.4"
```

Comment:
We need a newline at the end of the file. Please adjust your editor to enforce this.

Reply:
Done